README.md

```diff
@@ -48,7 +48,7 @@ public network (i.e. tagged VLAN).
 - Refer to the [Metal3 Setup Doc](./docs/setup/metal3-setup.md) for a walkthrough of the Metal3 Demo environment setup.
 - The [VBMH Setup Doc](./docs/setup/vbmh-setup.md) is a walkthrough of the setup of virtual machines to act as bare metal hosts.
 - The [RKE2 Setup Doc](./docs/setup/rke2-cluster.md) is a walkthrough of the deployment of a sample RKE2 cluster on the virtual bare metal hosts.
-- Example RKE2 deployment manifests exist [here](./docs/example-manifests/).
+- Example RKE2 deployment manifests exist [here](./sample-files/).
```

Comment:
Why do we move this to root? These manifests are tightly related to the setup guides so putting them under something like `/docs/setup/examples` should be more appropriate.